addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/config/LittleStation.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.openhab.binding.tankerkoenig.internal.config;
+
+/**
+ * The {@link LittleStation} class is the representing java model for the station specific json result of the prices
+ * request. (LittleStation because it has nearly no other information than gas prices. Stripped down version of
+ * Station.java)
+ *
+ * @author Dennis Dollinger
+ *
+ */
+
+public class LittleStation {
+
+    private double e5;
+    private double e10;
+    private double diesel;
+
+    private String status;
+
+    private String ID;
+
+    public double getE5() {
+        return e5;
+    }
+
+    public void setE5(double e5) {
+        this.e5 = e5;
+    }
+
+    public double getE10() {
+        return e10;
+    }
+
+    public void setE10(double e10) {
+        this.e10 = e10;
+    }
+
+    public double getDiesel() {
+        return diesel;
+    }
+
+    public void setDiesel(double diesel) {
+        this.diesel = diesel;
+    }
+
+    public String getStatus() {
+        return status;
+    }
+
+    public void setStatus(String status) {
+        this.status = status;
+    }
+
+    public String getID() {
+        return ID;
+    }
+
+    public void setID(String iD) {
+        ID = iD;
```

Comment:
I would not use different casing here. Better name the field (and parameter)  "id" like in other classes.